Pocket/pocket-ios
# fix(curatedInfo): pulling in old curatedInfo object for recomendations

## Summary
* Ensure that we follow how other clients choose to display item recommendation data, until this logic is fully upserted into the Pocket Graph. 

## References 
* https://getpocket.atlassian.net/browse/IN-748

## Implementation Details
* I followed the derivers implemented by the Web team [here](https://github.com/Pocket/web-client/blob/75f606321564cd85da82cc8ace0a8e14c799a48c/src/common/api/derivers/item.js#L268)
* I moved queries and fragments out of list.graphql into their own files. Once we have Apollo 1.0 setup and merged, this organization will get even more cleaner: https://github.com/Pocket/pocket-ios/pull/277

### Non-Goal
* Switching to Unified Home + Corpus Items

## Test Steps
* View Home
    * When home is viewed, and we have curated metadata present it is displayed.
    * You can verify correct metadata is shown by navigating to our curation tools and searching each recommendation in the corpus. When you pull up the item in the corpus, and the item has run on Firefox New Tab the metadata should match in the iOS app.

## PR Checklist:

## Screenshots
